website/static/css/style.css

```diff
@@ -692,3 +692,7 @@ button.close {
 div.ball-scale-blue>div {
     background-color: #BED1E2;
 }
+
+.ad_block_display{
```

Comment:
As per COSDev, use `lowercase-and-dashes` for CSS classes.
https://cosdev.readthedocs.io/en/latest/style_guides/html_css.html